__tests__/intergration.test.js

```diff
@@ -41,6 +41,43 @@ describe("GET: /api/topics", () => {
   });
 });
 
+//__________________ Section: /api/articles _______________//
+describe("GET: /api/articles/:article_id", () => {
+  test("200: responds with the specified article", () => {
+    const responseExpected = {
+      title: "Eight pug gifs that remind me of mitch",
+      topic: "mitch",
+      author: "icellusedkars",
+      body: "some gifs",
+      created_at: "2020-11-03T09:12:00.000Z",
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+        expect(article).toMatchObject(responseExpected);
+      });
+  });
+  test("404: sends appropriate status and msg when id does not exists", () => {
+    return request(app)
+      .get("/api/articles/9999999")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Article not found");
+      });
+  });
+});
+
```

Comment:
Nice job with this test, you can also test for what happens if you go to /api/articles/not-a-number